ws/javaee8/batch/joboperator-api/docs/index.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
+<html lang="en">
+<head>
+<!--
+
+    DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS HEADER.
+
+    Copyright (c) 1997-2017 Oracle and/or its affiliates. All rights reserved.

```

Comment:
Copyright year is not right. It should be:
Copyright (c) 2017 Oracle and/or its affiliates. All rights reserved.

Please fix all files